packages/forma-36-react-components/src/components/Button/Button.stories.tsx

```diff
@@ -29,9 +29,9 @@ storiesOf('Components|Button', module)
           'muted',
         )}
         size={select('size', [undefined, 'small', 'large'], undefined)}
-        loading={boolean('loading', false)}
-        indicateDropdown={boolean('indicateDropdown', false)}
-        disabled={boolean('disabled', false)}
+        isLoading={boolean('isLoading', false)}
+        hasDropdown={boolean('hasDropdown', false)}
+        isDisabled={boolean('isDisabled', false)}
```

Comment:
Wondering about `isFullWidth` below... Wouldn't `hasFullWidth` make more sense grammatically?

Reply:
@DanweDE I'd say no, `isFullWidth` is correct, and quite common. Think "the component is full-width" vs "the component has full-width" - the former (excuse the pun) makes more sense to me here.